subsys/net/ip/Kconfig.mgmt

```diff
@@ -30,7 +30,7 @@ config NET_MGMT_EVENT_STACK_SIZE
 config NET_MGMT_EVENT_QUEUE_SIZE
 	int "Size of event queue"
 	default 16 if NET_MGMT_EVENT_MONITOR
-	default 2
+	default 10
```

Comment:
> There is no reason why the default size should be 2,
because an event is only a combination of
{uint32_t + void*}

That's only true if event info is disabled, otherwise each individual event is several times larger. Still, I came to similar conclusion recently, that with increased number of net_mgmt users, 2 became a bit too low for robust operation. 10 might be a bit high, but since it's really an arbitrary value I guess it should be okeyish, worst case application can downsize it.

Reply:
Hmm, true. Missed that part.